IO/IniFileParser.cpp

```diff
@@ -365,9 +365,120 @@ bool IniFileParser::ParseHeader(TiXmlNode * xHeader)
                 _config->GetHostname().c_str(),
                 _config->GetPort());
         }
+
+        if(xTrajectories->FirstChild("optional_output")) {
+            Log->Write("WARNING: These optional options do only work with plain output format!");
```

Comment:
Please use the new logging mechanism

```suggestion
            Logging::Warning("These optional options do only work with plain output format!");
```
Don't forget to include `#include "general/Logger.h"`